### DISCUSSION TITLE
No ``define`` Function in ``ace.d.ts``, But Shows in Console. Why??

### DISCUSSION THREAD

**Discussion by KiddoV** [ajaxorg/ace#5023] [Q&A] [answered]
I don't see any function that defined for ``define``, but in the console I can see the ``define`` function?
What do I miss here? Am I losing my mind @@!??

```ts
import ace from 'ace-builds';

console.log(ace); //display below
```
![Capture](https://user-images.githubusercontent.com/28552977/212715050-8aa81672-89c9-40e8-9071-c042c0e5ecd3.PNG)

**Comment by nightwing** [comment, marked as answer]
`define` can be used only in ace-builds version, and is not a part of public api.

**Reply by KiddoV** [reply]
How do you call ``define function`` in TypeScript then? I want to make a custom Mode.
